roadmap.md

```diff
@@ -1,27 +1,29 @@
 # Roadmap
 
-Below is the current roadmap and timeline for RDSS, summer 2022 through January 2023. It is subject to change as needed, however it is written in markdown, so all history is recorded.
+Below is the current roadmap and timeline for RDSS. It is subject to change as needed, however it is written in markdown, so all history is recorded.
 
 ```mermaid
 gantt
-    title RDSS 2023
+    title RDSS Roadmap
     axisFormat  %Y-%m
     dateformat YYYY-MM
     section PDC
-    MVP software stabilizing: pd1,2023-05-01, 92d
     MVP testing and launch support: pd2,2023-07-31,62d
```

Comment:
I think this box need to be expanded.
Also should there be a separate box for migration?

Reply:
Good catch, done and done!